common/script/ops/scoreTask.js

```diff
@@ -90,79 +109,80 @@ function _gainMP (user, val) {
   }
 }
 
-// HP modifier
-// ===== CONSTITUTION =====
-// TODO Decreases HP loss from bad habits / missed dailies by 0.5% per point.
+// Decrease user's health when:
+// - user does negative Habits
+// - cron finds uncompleted Dailies
 function _subtractPoints (user, task, stats, delta) {
-  let conBonus = 1 - user._statsComputed.con / 250;
+  let conBonus = 1 - user._statsComputed.con / 250; // CON decreases HP loss
```

Comment:
Also, changing this will make it easy to pull out the bonus-per-point values for con, int, and per and add them as `consts` at the top of the file.